redpanda/connect/values.yaml

```diff
@@ -0,0 +1,77 @@
+# Default values for redpanda-connect.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+connect:
+  # root-config: ""
+  configs: "configs/*.yaml"
+
+image:
+  repository: docker.redpanda.com/redpandadata/redpanda
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is "latest".
+  tag: "v24.1.6" # latest currently does not support rpk connect yet
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
```

Comment:
Lets move to helm-values that only contain the diffs to the defaults , in the future , please. Its fine right now as this is experimental, but lets keep that in mind, please.